netci.groovy

```diff
@@ -198,6 +198,27 @@ commitPullList.each { isPr ->
   addRoslynJob(myJob, jobName, branchName, isPr, triggerPhraseExtra, triggerPhraseOnly)
 }
 
+// Open Integration Tests
+commitPullList.each { isPr ->
+  def jobName = Utilities.getFullJobName(projectName, "open-vsi", isPr)
+  def myJob = job(jobName) {
+    description('open integration tests')
+    label('auto-win2012-20160912')
+    steps {
+      batchFile("""set TEMP=%WORKSPACE%\\Binaries\\Temp
+mkdir %TEMP%
+set TMP=%TEMP%
+set VS150COMNTOOLS=%ProgramFiles(x86)%\\Microsoft Visual Studio\\VS15Preview\\Common7\\Tools\\
```

Comment:
Because the VS environment variable doesn't exist in Dev15 Preview 4 and we haven't implemented the general build level logic to detect where VS is installed yet.